Us 28 login log

FlorenzoC opened the pull request:
First login log

FlorenzoC committed:
Added LoginLog, start getting device and date

Vemka committed:
login_log back v0.1

it just works!

Vemka committed:
login logs front

simple version

Vemka committed:
better display front

better logs page

Vemka committed:
removed unused vars

Commit:
Merge branch 'SP4' into US-28-Login-Log

ifrins reviewed (commented):
S'haurien de fer tests abans de fer merge.

FlorenzoC committed:
Added login logs test

Es contemplen els logs success i els logs failed on el correu és àlid (per evitar robos de cuenta)

FlorenzoC committed:
Merge branch 'SP4' into US-28-Login-Log

dabytmc reviewed (changes requested):
Has de posar el test abans del shutdown API, sino la API esta apagada quan intentes fer el test.

FlorenzoC committed:
Get login log before shutdown api

Commit:
Added tests and corrected one exception.

Commit:
Merge branch 'SP4' into US-28-Login-Log
